src/components/common/item-selector/typeahead.js

```diff
@@ -138,13 +191,21 @@ export default class Typeahead extends Component {
     resultsTruncatedMessage: null
   };
 
+  static getDerivedStateFromProps(props, state) {
+    //  invoked after a component is instantiated as well as before it is re-rendered
+    const searchResults = getOptionsForValue(state.entryValue, props, state);
+
+    return {searchResults};
+  }
+
   constructor(props) {
     super(props);
 
     this.state = {
-      searchResults: this.getOptionsForValue(
+      searchResults: getOptionsForValue(
```

Comment:
Surely not necessary to do this here now since it is handled by `getDerivedStateFromProps`?

Reply:
👍